src/pages/wallet/rns/domainToAddress.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+import Resolver from '@rsksmart/rns-resolver.js';
+
+/**
+ * Convert a RSK domain to address if available
+ * @param {string} domain string Domain to be resolved
+ * @param {string} symbol chainId to get address for
+ * @param {type} constant: Mainnet or Testnet
+ */
+export const domainToAddress = (domain, symbol, type) => {
+  // eslint-disable-next-line new-cap
```

Comment:
What does this comment mean?

Reply:
The linter didn't like line 12:
```
const resolver = type === 'Mainnet' ? new Resolver.forRskMainnet() : new Resolver.forRskTestnet();
```

It wanted the `f` in `forRskMainnet` and `forRskTestnet` to be capitalized.